{"name": "Проверка ответа при запросе на получение списка заказов неаутентифицированного пользователя", "status": "passed", "start": 1719121817830, "stop": 1719121818033, "uuid": "9a12d82f-1911-42ba-99b3-2b8a1902aadd", "historyId": "d932a420574a8af1ba53e8754c88e341", "testCaseId": "d932a420574a8af1ba53e8754c88e341", "fullName": "tests.test_get_user_orders.TestGetOrders#test_get_orders_unauthenticated_user_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_user_orders"}, {"name": "subSuite", "value": "TestGetOrders"}, {"name": "host", "value": "LAPTOP-0911SMBB"}, {"name": "thread", "value": "32628-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_user_orders"}]}